README.adoc

```diff
@@ -114,15 +114,15 @@ Both sites will be maintained to promote the development and use of MicroProfile
 The https://wiki.eclipse.org/MicroProfile[MicroProfile wiki] contains up-to-date information about the project development, tools and other documentation.
 
 === Google Group
-The link:https://groups.google.com/forum/#!forum/microprofile[Microprofile Google Group] is the preferred place for discussions to take place.
+The link:https://groups.google.com/forum/#!forum/microprofile[MicroProfile Google Group] is the preferred place for discussions to take place.
 
 === Mailing Lists
-Besides the Google Group threads (main source of communication), there is also an link:https://dev.eclipse.org/mailman/listinfo/microprofile-dev[Eclipse microprofile-dev mailing list] which should be used for conversation specific to the Eclipse way of doing things.
+Besides the Google Group threads (main source of communication), there is also an link:https://dev.eclipse.org/mailman/listinfo/microprofile-dev[microprofile-dev mailing list] which should be used for conversation specific to the Eclipse way of doing things.
 By using this mailing list, the Eclipse team can more quickly recognize and respond to our requests. +
 
 Since the link:https://projects.eclipse.org/projects/technology.microprofile[MicroProfile project] is still in incubation, another useful mailing list is the link:https://dev.eclipse.org/mailman/listinfo/incubation[incubation mailing list] for asking general Eclipse process questions.
 
 == Notable Contributors
 There is a very large number of people and vendors already involved in the link:https://projects.eclipse.org/projects/technology.microprofile[MicroProfile project].
 The most complete list of contributors can be found at our link:https://microprofile.io/contributors[microprofile.io site].
```

Comment:
Is it the most complete?  How is this "contributors" web site kept up to date?  Is it automated?  How does one get listed on this page?  The other list referenced in line 128 is only the committers (not all contributors).  That list below is automated.

Reply:
Currently, Summers is working on fixing with the MP Projects' page which will lead to updating the Contributors page as well. Using git the updates will continue to be automatic by activity, yet there are many broken items which require Summer's help. See MP thread https://groups.google.com/forum/#!topic/microprofile/7xEB4PO9Zqs

Reply:
Fixed